src/sentry/static/sentry/app/components/group/sentryAppExternalIssueForm.jsx

```diff
@@ -29,6 +32,47 @@ class SentryAppExternalIssueForm extends React.Component {
     addErrorMessage(t('Unable to %s %s issue.', action, appName));
   };
 
+  getOptions = (field, input) => {
+    return new Promise(resolve => {
+      this.debouncedOptionLoad(field, input, resolve);
+    });
+  };
+
+  debouncedOptionLoad = debounce(
```

Comment:
Is "debounce" a common term used for this? I know it's from `lodash`, but I've never heard of it before. Do you think it's worth it to name it something else, like `requestEveryN` or `requestAfterWait` or something?

Reply:
Added a comment explaining its use